android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.nt4f04und.sweyer">
```

Comment:
It is really not needed?
upd: checked, docs conifrm it's needed to remove it https://developer.android.com/guide/topics/manifest/manifest-element#package

Reply:
Yeah, this is now set in the gradle file.